api/src/utils/user-utils.ts

```diff
@@ -0,0 +1,58 @@
+import { difference } from 'lodash';
+import { isAdmin } from './auth-utils';
+import User from '../models/user';
+import { SessionUser } from './helpers';
+
+// All fields in user
+const allFields = Object.keys(User.schema.paths);
+
+const nonViewableFields = ['oauthID', '__v'];
+
+const nonEditableFields = ['_id', 'oauthID', '__v'];
+// Fields that non-Admins and other users cannot view
+const privateFields = ['phone'];
```

Comment:
I think `private` is also a little ambiguous - how about `adminViewableFields`?

Reply:
I agree